<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1098</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(20, 20, 20, 150);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>1078</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBotton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-3</y>
       <width>1090</width>
       <height>771</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
border-style: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>1081</width>
       <height>761</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,0" columnstretch="1,1,1">
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="4,1" columnstretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>finishedImage.gif</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,1,7">
          <item row="0" column="0">
           <widget class="QLabel" name="emptybox3">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(46, 46, 140, 150);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(46, 46, 140, 150);</string>
            </property>
            <property name="text">
             <string>seconds to complete</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="timetobenchmark">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(46, 46, 140, 150);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="emtpybox1">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="emtpybox2">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>label</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>closeBotton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
